doc/reference/cli.rst

```diff
@@ -0,0 +1,187 @@
+Command-Line Interface
+======================
+
+This is a short overview of the commands available in ``dune``. Reference
+documentation for each command is available through ``dune COMMAND --help``.
+
+.. describe:: dune build
+
+   Build the given targets, or the default ones.
+
+.. describe:: dune cache
+
+   Manage the shared cache of build artifacts.
+
+   .. describe:: dune cache size
+
+      Query the size of the Dune cache.
+
+   .. describe:: dune cache trim
+
+      Trim the dune cache.
+
+.. describe:: dune clean
+
+   Clean the project.
+
+.. describe:: dune coq
+
+   Command group related to Coq.
+
+   .. describe:: dune coq top
+
+      Execute a Coq toplevel with the local configuration.
+
+.. describe:: dune describe
+
+   Describe the workspace.
+
+.. describe:: dune diagnostics
+
+   Fetch and return errors from the current build.
+
+.. describe:: dune exec
+
+   Execute a command in a similar environment as if installation was performed.
+
+.. describe:: dune fmt
+
+   Format source code.
+
+.. describe:: dune format-dune-file
+
+   Format ``dune`` files.
+
+.. describe:: dune help
+
+   Additional Dune help.
+
+.. describe:: dune init
+
+   Command group for initializing Dune components.
+
+   .. describe:: dune init executable
+
+      Initialize a binary executable.
+
+   .. describe:: dune init library
+
+      Initialize an OCaml library.
+
+   .. describe:: dune init project
+
+      Initialize a whole OCaml project.
+
+   .. describe:: dune init test
+
+      Initialize a test harness.
+
+.. describe:: dune install
+
+   Install packages defined in workspace.
+
+.. describe:: dune installed-libraries
+
+   Print out libraries installed on the system.
+
+.. describe:: dune ocaml
+
+   Command group related to OCaml.
+
+   .. describe:: dune ocaml dump-dot-merlin
+
+      Print Merlin configuration.
+
+   .. describe:: dune ocaml merlin
+
+      Command group related to merlin.
+
+      .. describe:: dune ocaml merlin dump-config
+
+         Prints the entire content of the merlin configuration for the given
+         folder in a user friendly form.
+
+      .. describe:: dune ocaml merlin start-session
+
+         Start a merlin configuration server.
+
+   .. describe:: dune ocaml ocaml-merlin
+
+      Start a merlin configuration server.
+
+   .. describe:: dune ocaml top
+
+      Print a list of toplevel directives for including directories and loading
+      cma files.
+
+   .. describe:: dune ocaml top-module
+
+      Print a list of toplevel directives for loading a module into the topevel.
+
+   .. describe:: dune ocaml utop
+
+      Load library in utop.
+
+.. describe:: dune ocaml-merlin
+
+   Start a merlin configuration server.
```

Comment:
```suggestion
         Start a Merlin configuration server.

   .. describe:: dune ocaml ocaml-merlin

      Start a Merlin configuration server.

   .. describe:: dune ocaml top

      Print a list of toplevel directives for including directories and loading
      CMA files.

   .. describe:: dune ocaml top-module

      Print a list of toplevel directives for loading a module into the toplevel.

   .. describe:: dune ocaml utop

      Load library in UTop.

.. describe:: dune ocaml-merlin

   Start a Merlin configuration server.
```

Capitalizations & a type all in one comment/suggestion for ease of accepting.

Reply:
I think `cma` -> `.cma`

Reply:
Cool! Then in monospace!